scripts/globals/promyvion.lua

```diff
@@ -13,39 +14,45 @@ tpz.promyvion = tpz.promyvion or {}
 ------------------------------------
 
 local function maxFloor(ID)
-    local m = 0
+	if ENABLE_COP == 1 then
+		local m = 0
 
-    for _, v in pairs(ID.mob.MEMORY_RECEPTACLES) do
-        m = math.max(m, v[1])
-    end
+		for _, v in pairs(ID.mob.MEMORY_RECEPTACLES) do
+			m = math.max(m, v[1])
+		end
```

Comment:
Lot o' tab indents being used in these changed files.  All tabs need to be replaced with 4 spaces.

Notepad++ has this as a settings option so that when you press tab, it gives you 4 spaces instead.

Reply:
My mistake, I'll fix that.